### DISCUSSION TITLE
recursive_will_execute has been patched

### DISCUSSION THREAD

**Discussion by Davikar** [rgthree/rgthree-comfy#158] [General]
The recursive_will_execute function has been patched in ComfyUI. Not sure how this compares to your implementation or if it causes any compatibility issues:

https://github.com/comfyanonymous/ComfyUI/commit/f81dbe26e2e363c28ad043db67b59c11bb33f446

**Comment by rgthree** [comment]
Ah, nice to see this finally getting addressed. It doesn't quite solve the full issue, so you're still better off keeping the optimization from rgthree-comfy (and, luckily, this change doesn't break it).

I'll reach out there and see if we can get a proper fix.
